engine/consensus/matching/request_tracker.go

```diff
@@ -0,0 +1,33 @@
+package matching
+
+import "github.com/onflow/flow-go/model/flow"
+
+type RequestTracker struct {
```

Comment:
May I ask to please include a minimal goDoc for the `RequestTracker` type as well as each of its public methods. 🙏 🙇